modules/oadp-restore-request-limit.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: CONCEPT
+[id="oadp-restore-request-limit_{context}"]
+= Pods crash or restart due to lack of memory or CPU
```

Comment:
@RichardHoch
We always make module IDs identical or similar to their headings. 
Also, neither of them seems suitable here. The heading looks like a known issue in RN. And the ID includes 'limit', whereas the module itself does not mention any limits. This module is about setting requests for additional resources, so why not title it accordingly? I would suggest:
id="oadp-setting-resource-requests-for-memory-and-cpu_{context}" 
=Setting resource requests for memory and CPU

Reply:
Module IDs are identical to the **file name** (without ".adoc."), not the **heading.**  https://redhat-documentation.github.io/modular-docs/  **Be sure your IDs are identical to the file name,** regardless of the heading name.
The purpose of his Jira is to incorporate some information into the Troubleshooting section, since it only applies in the event of a pod crash; it is intentionally not part of the usual configuration process.  
I have renamed the module to better reflect its location. 

You're right that "Known Issues" and items in a "Troubleshooting" section look similar, but this is not a known issue -- the solution performs as it should.

Reply:
@RichardHoch 
Yes, you are right - IDs should be identical to file names, not to headings. So how did you rename the file?

Re known issue: That was my point - this module is not a known issue, it's a solution, so it should be titled differently. By the way, why is it not a procedure?

Reply:
Oh, I see - oadp-pod-crash-set-resource-request.adoc  Now it makes sense.